recipes/toolchain/deactivate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Strictly speaking, I don't think we really need to deactivate. It isn't as if we are putting the environment back into the state we started with anyway (some of these may have been defined before activation).